### PR TITLE
Heron on Kubernetes docs

### PR DESCRIPTION
**Pull request by lucperkins**

### CODE REVIEW

File: website/content/snippets/heron-on-kubernetes-config.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+* `heron.kubernetes.scheduler.uri` --- Specifies the URI for the Kubernetes API service. If you're 
+using `kubectl proxy` to access your API, this will need to be set to `http://localhost:8001` if 
+you're using the standard proxy configuration.
+
+* `heron.scheduler.is.service` --- This config indicates whether the scheduler
+is a service. In the case of Kubernetes, it should be set to `False`.
+
+* `heron.executor.docker.image` --- Specified the Docker image to be used for running the executor.
+To use the Docker image created for the latest release, you can set this value to 
+`streamlio/heron:<version>-<os>`. Example: `streamlio/heron:0.14.8-ubuntu14.04`
```
Comment (cckellogg) [inline website/content/snippets/heron-on-kubernetes-config.md]: let's update the image to be heron/heron:<version> or heron/heron:latest

File: website/content/snippets/heron-on-kubernetes-config.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+heron.directory.home:                       $HERON_HOME
+
+# The URI of kubernetes API
+heron.kubernetes.scheduler.uri: "http://localhost:8001"
+
+# Invoke the IScheduler as a library directly
+heron.scheduler.is.service:                  False
+
+# docker repo for executor
+heron.executor.docker.image: 'streamlio/heron:0.14.8-ubuntu14.04'
```
Comment (cckellogg) [inline website/content/snippets/heron-on-kubernetes-config.md]: Let's update this to a heron image heron/heron:latest

File: website/content/docs/operators/deployment/schedulers/kubernetes.md [hunk 3]
```diff
@@ -1,189 +1,286 @@
...
+
+* Locally using [Minikube](#minikube)
+* In the cloud on [Google Container Engine](#google-container-engine) (GKE)
+* In [any other](#general-kubernetes-clusters) Kubernetes cluster
+
+## Requirements
+
+In order to run Heron on Kubernetes, you will need:
+
+* A Kubernetes cluster with at least 3 nodes
```
Comment (cckellogg) [inline website/content/docs/operators/deployment/schedulers/kubernetes.md]: should we add or Minikube here?
A Kubernetes cluster with at least 3 nodes or Minikube

File: website/content/snippets/heron-on-kubernetes-config.md (hunk 2 above)
Comment (lucperkins) [reply website/content/snippets/heron-on-kubernetes-config.md]: This is just a WIP for now. I'll ping you when it's ready for review.

**Pull request merged**